holidays/countries/cape_verde.py

```diff
@@ -0,0 +1,105 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.constants import OPTIONAL, PUBLIC
+from holidays.groups import ChristianHolidays, InternationalHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class CapeVerde(HolidayBase, ChristianHolidays, InternationalHolidays):
+    """Cape Verde holidays.
+
+    References:
+        * [Public holidays in Cape Verde] <https://en.wikipedia.org/wiki/Public_holidays_in_Cape_Verde>
+        * [Today's and Upcoming Holidays in Cape Verde] <https://www.timeanddate.com/holidays/cape-verde/>
+        * [Democracy Day] <https://www.officeholidays.com/holidays/cape-verde/cape-verde-democracy-day>
+        * [National Heroes Day] <https://www.officeholidays.com/holidays/cape-verde/guinea-bissau-national-heroes-day>
+    """
+
+    country = "CV"
+    default_language = "pt_PT"
+    supported_languages = ("en_US", "pt_PT", "uk")
```

Comment:
I recommend avoiding unnecessary additions of other languages unless they are listed in the official source, have an existing translation source, or are languages you personally speak:
```suggestion
    supported_languages = ("en_US", "pt_PT")
```